dcc-portal-ui/app/styles/styles.scss

```diff
@@ -411,6 +412,9 @@ a.no-style {
   }
   i {
     height: 1em;
+    display: flex;
```

Comment:
IIRC, We've had issues with flex on some browsers. How widely supported is this?